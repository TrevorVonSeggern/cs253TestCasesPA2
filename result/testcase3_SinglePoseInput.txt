0 0 0 -0.00416938 0.354598 0.0167151 -0.00790055 0.701007 0.0191316 -0.0105299 0.871262 0.0116944 -0.19874 0.601166 -0.00220366 -0.450043 0.580583 0.00609765 -0.682626 0.596308 -0.0179295 -0.741812 0.611333 -0.0135724 0.174876 0.596822 0.00773782 0.454287 0.584589 0.0332173 0.681261 0.589473 0.0229755 0.782642 0.599251 0.0422573 -0.0838508 0.000575605 -0.0433715 -0.130115 -0.468273 -0.0332298 -0.142072 -0.904543 -0.0111558 -0.15125 -1 -0.0959453 0.086067 -0.00140205 -0.0432588 0.126496 -0.466407 -0.0160139 0.145264 -0.902639 0.0182676 0.147206 -0.994216 -0.0762759 -0.00704913 0.615565 0.02121 -0.843693 0.617131 -0.0401913 -0.690039 0.614238 -0.0441729 0.859418 0.596359 0.0227626 0.852719 0.598463 -0.00316764 
