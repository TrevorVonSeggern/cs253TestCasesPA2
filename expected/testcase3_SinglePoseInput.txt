0 0 0 -0.00403223 0.342934 0.0161653 -0.00764067 0.677948 0.0185023 -0.0101835 0.842603 0.0113097 -0.192203 0.581392 -0.00213117 -0.43524 0.561486 0.00589708 -0.660172 0.576694 -0.0173398 -0.717411 0.591225 -0.013126 0.169124 0.577191 0.0074833 0.439344 0.56536 0.0321247 0.658852 0.570083 0.0222197 0.756898 0.579539 0.0408673 -0.0810927 0.000556671 -0.0419449 -0.125835 -0.45287 -0.0321368 -0.137398 -0.874789 -0.0107889 -0.146275 -0.967107 -0.0927894 0.0832359 -0.00135593 -0.0418359 0.122335 -0.451065 -0.0154872 0.140486 -0.872948 0.0176668 0.142364 -0.961513 -0.0737669 -0.00681726 0.595317 0.0205123 -0.815941 0.596831 -0.0388692 -0.667341 0.594034 -0.0427199 0.831149 0.576743 0.0220138 0.82467 0.578777 -0.00306345 
